### ISSUE TITLE
A Question of VQA Validation Set Data

### ISSUE THREAD

**Issue by waxnkw** [OFA-Sys/OFA#173]
Thanks for your excellent code. It seems like that the VQA validation set data (vqa_val.tsv) you provide only contains an image id without a question id. I do not know whether I understand correctly. I attach a screenshot of the TSV file. The first number and the second number is the same, while for vqa_test.tsv is different.
![Selection_657](https://user-images.githubusercontent.com/34906503/179973660-1a59a6b4-a6c3-46c3-9def-a36cb4112eb0.png)

**Comment by yangapku** [comment]
Yes, you are right, but it does not affect the training and validation process. The 1st-indexed image-id column in the TSV file is actually not used by the VQA finetuning & evaluation scripts (notice the specified `selected_cols` in the `train_vqa_distributed.sh`, `evaluate_vqa_beam.sh` and `evaluate_vqa_allcand.sh` does not include the 1st-indexed column, which stands for the field of image-id). For the question-id column, it is only actually used to obtain the result file on test-set for submitting to the official VQA evaluation server, thus the training and validation process will not be affected if it is mis-specified.

**Comment by waxnkw** [comment]
Got it. Thanks very much for your reply! I am trying to analyze the wrong samples of OFA on VQA dataset. I am wondering whether it is convenient to release the generated results (**val_predict.json and test_predict.json**) of OFA-Huge using **all_cand** method on VQA dataset. If you can release it, it will be very helpful!

**Comment by yangapku** [comment]
Hi, of course. Please download this file `test-dev_predict_ofa-huge.json` ([link](https://ofa-beijing.oss-cn-beijing.aliyuncs.com/datasets/vqa_data/test-dev_predict_ofa-huge.json)). This file contains the all-candidate inference result on VQA **test-dev** set for OFA-huge model, which can achieve the 82.00 test-dev accuracy (yes/no: 94.65, number: 71.96, other: 73.43).

**Comment by waxnkw** [comment]
Thanks so much! If possible, can I also get the results on **test-std** and **val**?

**Comment by yangapku** [comment]
Hi, I'm sorry currently we are not planning to release the inference result on VQA test-std, in case someone who may never have run our model could still directly submits the prediction file and get leading performance on the leaderboard, which maybe unfair.

**Comment by waxnkw** [comment]
Got it. Thanks!
